heron/schedulers/src/java/com/twitter/heron/scheduler/kubernetes/KubernetesScheduler.java

```diff
@@ -52,9 +60,12 @@ protected KubernetesController getController() {
 
   @Override
   public void initialize(Config aConfig, Config aRuntime) {
+    LOG.log(Level.INFO, aRuntime.toString());
```

Comment:
In the past we've had issues with credentials being logged when doing this. Can we change this to FINE or remove it. If we keep it, it's also good to add some sort of contextual logging to the message.